dogstatsd/entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#set -e
+
+if [[ $DD_API_KEY ]]; then
```

Comment:
By using supervisor you can reuse the same entry point as the agent i believe. It would be easier to maintain